### ISSUE TITLE
App screenshots 

### ISSUE THREAD

**Issue by Donnnno** [accrescent/meta#16]
It would be nice for the user & dev to have some screenshots in the app so that you can see what you install.

**Comment by matchboxbananasynergy** [comment]
Agreed. It would be great if we could set some criteria that would ensure that the screenshots are crisp and look great, but I'm not sure what that criteria would look like.

I've just noticed that in other stores, screenshots can be very inconsistent from app to app in terms of quality etc.

Accrescent has a chance to do this right from the start so that the overall UX of the store is that much better.

**Comment by matchboxbananasynergy** [comment]
Accrescent can potentially support separate screenshot sections for phone and large screen (tablet/foldable) devices, where developers can put the appropriate screenshots in each.

Optionally, Accrescent can detect the screen size and default to the appropriate section by default.
